### PR TITLE
feat(cart): add place order facade fields

### PR DESCRIPTION
**Pull request by griest024**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/graycoreio/daffodil/blob/master/CONTRIBUTING.md#commit
- [x] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

```
[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[ ] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Fixes: N/A

## Does this PR introduce a breaking change?
```
[ ] Yes
[ ] No
```

## Other information

### CODE REVIEW

File: libs/cart/testing/src/helpers/mock-cart-facade.ts [hunk 1]
```diff
@@ -35,5 +36,10 @@ export class MockDaffCartFacade implements DaffCartFacadeInterface {
   availablePaymentMethods$: BehaviorSubject<DaffCart['available_payment_methods']> = new BehaviorSubject([]);
   isCartEmpty$: BehaviorSubject<boolean> = new BehaviorSubject(true);
 
+  orderResultLoading$ = new BehaviorSubject<boolean>(false);
```
[the remark this answers is not in the document]
Comment (griest024) [reply libs/cart/testing/src/helpers/mock-cart-facade.ts:39]: Yeah I think so. VS Code intellisense shows the correct type.

[the remark this answers is not in the document]
Comment (lderrickable) [reply libs/cart/testing/src/helpers/mock-cart-facade.ts:39]: The last two are coming in as `BehaviorSubject<any>` for me. Could you add the types for these?

[the remark this answers is not in the document]
Comment (griest024) [reply libs/cart/testing/src/helpers/mock-cart-facade.ts:39]: hm for me they are typed.

**Comment by lderrickable** [comment]
lgtm

**Pull request merged**